scripts/dev/k8s_conditions.py

```diff
@@ -45,24 +47,51 @@ def wait(
     return False
 
 
-def _ignore_error_codes(fn, codes):
+def call_eventually_succeeds(
+    fn: Callable,
+    sleep_time: float = SLEEP_TIME,
+    timeout: int = INFINITY,
+    exceptions_to_ignore: Optional[Tuple[BaseException]] = None,
+) -> bool:
+    """
+    call_eventually_succeeds is similar to wait but for when we don't care about the returned value of fn
+    (it can even not return anything).
```

Comment:
it looks like we never return anything, maybe we can update the comment to say the return value isn't examined at all

Reply:
Yes poor phrasing in the comment!